src/data/reducers/authentication.js

```diff
@@ -43,6 +44,12 @@ const authentication = (state = {
         error: null,
         email: null,
       };
+    case LOCAL_USER_LOADED:
+      return {
```

Comment:
This is basically duplicated from FETCH_LOGIN_SUCCESS, other than the loading flag.

We could combine the two cases but I didn't know if the fall-through would be more confusing to read.